setup.py

```diff
@@ -121,11 +123,18 @@ def find_packages(path, base=""):
     "jsonschema>=2.3.0",
     "netifaces>=0.10.6",
     "websocket-client>=0.18.0",
-    "pybonjour>=1.1.1",
     "ujson>=1.33",
     "wsaccel>=0.6.2"
 ]
 
+# Check if setuptools is recent enough to recognise python_version syntax
```

Comment:
A comment as to why any of this is needed is probably useful here, to call out that pybonjour doesn't support Python3

Reply:
Will do